test/integration_test/specs/kubevirt-templates/fedora/fedora-pvc-template-wait-first-consumer.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: fedora-template-pvc-wait-first-consumer
+  namespace: openshift-virtualization-os-images
+  labels:
+    app: containerized-data-importer
+  annotations:
+    cdi.kubevirt.io/storage.bind.immediate.requested: "true"
+    cdi.kubevirt.io/storage.import.endpoint: https://ftp-nyc.osuosl.org/pub/fedora/linux/releases/37/Cloud/x86_64/images/Fedora-Cloud-Base-37-1.7.x86_64.qcow2
+spec:
+  storageClassName: sc-sharedv4svc-nolock-wait-first-consumer
+  accessModes:
+  - ReadWriteMany
+  resources:
+    requests:
+      storage: 50Gi
```

Comment:
We don't need this file. Template PVC will never use WFFC. Here, you are overriding it with the annotation anyways.

Reply:
Removed.